### DISCUSSION TITLE
Possible to use without Wlan?

### DISCUSSION THREAD

**Discussion by lBulgur** [jomjol/AI-on-the-edge-device#611] [Q&A] [answered]
Hi, I would like to use the device to record the meter data at different locations without a WiFi connection on the SD card. Is that possible? - Perhaps with a temporary wireless connection via smartphone for the installation?

**Comment by jomjol** [comment, marked as answer]
Technically possible, but not implemented. If anybody has time to implement, please let me know.

**Reply by al3xt3r** [reply]
The use without access to wifi would be most desirable as in many houses the meters are in the basement where no wifi connection is available. I have the same problem.

In my case I might be able to use the neighbours wifi. In this case it would be nice to be able to upload the data to a cloud or email it regularly.

**Reply by abdullrahmanesmael07** [reply]
what

**Comment by pos-ei-don** [comment]
Upload to cloud is possible using mqtt.
